Unable to build Docker images

### Actual behavior

Building docker images for Lisk Service fails with the following error because of the removal of the pinned apk package versions from the alpine package repository. 

```
Use 'docker scan' to run Snyk tests against images to find vulnerabilities and learn how to fix them
cd ./services/blockchain-connector && docker build --tag=lisk/service_blockchain_connector ./
[+] Building 6.4s (9/13)                                                                                                                                                                              
 => [internal] load build definition from Dockerfile                                                                                                                                             0.0s
 => => transferring dockerfile: 651B                                                                                                                                                             0.0s
 => [internal] load .dockerignore                                                                                                                                                                0.0s
 => => transferring context: 872B                                                                                                                                                                0.0s
 => [internal] load metadata for docker.io/library/node:16-alpine                                                                                                                                1.0s
 => [internal] load build context                                                                                                                                                                0.0s
 => => transferring context: 867.86kB                                                                                                                                                            0.0s
 => CACHED [builder 1/5] FROM docker.io/library/node:16-alpine@sha256:1621ddffc775ccf10dc8710660e70d5bfb8e4ee349b926f133698e6512144b73                                                           0.0s
 => [stage-1 2/5] RUN adduser -D lisk &&     chown -R lisk:lisk /home/lisk/                                                                                                                      0.3s
 => ERROR [builder 2/5] RUN apk add --no-cache alpine-sdk=1.0-r1 python3=3.10.9-r1 cmake=3.24.3-r0 &&     adduser -D builder &&     mkdir /home/builder/build &&     chown -R builder:builder /  5.3s
 => [stage-1 3/5] WORKDIR /home/lisk/lisk-service/blockchain-connector                                                                                                                           0.0s
 => [stage-1 4/5] COPY ./ .                                                                                                                                                                      0.0s
------                                                                                                                                                                                                
 > [builder 2/5] RUN apk add --no-cache alpine-sdk=1.0-r1 python3=3.10.9-r1 cmake=3.24.3-r0 &&     adduser -D builder &&     mkdir /home/builder/build &&     chown -R builder:builder /home/builder/:
#9 0.264 fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/aarch64/APKINDEX.tar.gz
#9 2.288 fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/aarch64/APKINDEX.tar.gz
#9 5.312 ERROR: unable to select packages:
#9 5.331   python3-3.10.10-r0:
#9 5.331     breaks: world[python3=3.10.9-r1]
------
executor failed running [/bin/sh -c apk add --no-cache alpine-sdk=1.0-r1 python3=3.10.9-r1 cmake=3.24.3-r0 &&     adduser -D builder &&     mkdir /home/builder/build &&     chown -R builder:builder /home/builder/]: exit code: 1
make: *** [build-connector] Error 1
sameer@Sameers-MacBook-Pro lisk-service % 
```

### Expected behavior

Building the docker images should not fail.

### Steps to reproduce

- Checkout Lisk Service `v0.7.0-alpha.6` and run `make build`

### Which version(s) does this affect? (Environment, OS, etc...)

Lisk Service `v0.7.0-alpha.6`

--- a/services/blockchain-connector/Dockerfile
+++ b/services/blockchain-connector/Dockerfile
@@ -2,7 +2,7 @@
 
 FROM node:16-alpine AS builder
 
-RUN apk add --no-cache alpine-sdk=1.0-r1 python3=3.10.9-r1 cmake=3.24.3-r0 && \
+RUN apk add --no-cache alpine-sdk python3 cmake && \
     adduser -D builder && \
     mkdir /home/builder/build && \
     chown -R builder:builder /home/builder/

